projects/WTI-API/websockets/WebsocketEnums/WebsocketMsgType.java

```diff
@@ -1,5 +1,5 @@
 package WebsocketEnums;
 
 public enum WebsocketMsgType {
-	UNDEFINED, JUDGED, CONTEST_CLOCK, TEST, CLARIFICATION, STANDINGS, CONNECTION_DROPPED
+	UNDEFINED, JUDGED, CONTEST_CLOCK, TEST, CLARIFICATION, CLARIFICATION_ANNOUNCEMENT, STANDINGS, CONNECTION_DROPPED
```

Comment:
I suggest `CLARIFICATION_ANNOUNCEMENT` be renamed to just `ANNOUNCEMENT`.  See additional comments in following files for some explanation of why I think that makes more sense.